addons/mail/static/tests/message/message_reply.test.js

```diff
@@ -120,3 +121,36 @@ test("click on message in reply highlights original message", async () => {
     );
     await contains(".o-mail-Message.o-highlighted:contains('This message has been removed')");
 });
+
+test("reply action functionality in Chatter", async () => {
```

Comment:
Would name it _"Can reply to message in Chatter"_.

Good test name are natural to read and ideally be so good that you don't have to read the code to get the functional intent of the test.

So test names that are too broads are not good.
We have unfortunately some bad examples, but we try our best to keep good name for newer tests because this contributes to more maintainable tests.